Orange/tests/test_softmax_regression.py

```diff
@@ -0,0 +1,34 @@
+import unittest
+
+import Orange
+from Orange.classification import SoftmaxRegressionLearner
+
+
+class SoftmaxRegressionTest(unittest.TestCase):
+    def test_SoftmaxRegression(self):
+        table = Orange.data.Table('iris')
+        learner = SoftmaxRegressionLearner()
+        results = Orange.evaluation.CrossValidation(table, [learner], k=10)
```

Comment:
When using cross-validation in unit tests it is usually enough to use k=3 (or even k=2) which makes it about 3x faster.